az-digital/az_quickstart
# Closes #2710 Remove Arizona Bootstrap overrides.

## Description
Now that https://github.com/az-digital/arizona-bootstrap/pull/901 was merged and is part of Arizona Bootstrap v2.0.25, we don't need this anymore

## Related issues
Closes #2710

## How to test
Ensure mobile browsers still have correct styles when mobile nav exists on page.  (no horizontal scroll)

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
